docs/systemuicontroller.md

```diff
@@ -0,0 +1,82 @@
+# System UI Controller for Jetpack Compose
+
+[![Maven Central](https://img.shields.io/maven-central/v/com.google.accompanist/accompanist-systemuicontroller)](https://search.maven.org/search?q=g:com.google.accompanist)
+
+System UI Controller provides easy-to-use utilities for updating the System UI bar colors within Jetpack Compose.
+
+## Usage
+To setup Insets in your composables, you need to call the provide an instance to the `LocalSystemUiController`
+composition local. This would typically be done near the top level of your composable hierarchy:
+
+``` kotlin
+setContent {
+    // Create a controller, and provide it to the LocalSystemUiController
+    val controller = androidSystemUiController(LocalView.current)
```

Comment:
This should be remembered right? Or should we offer a `rememberAndroidSBC` method?